README.md

```diff
@@ -1,9 +1,228 @@
-# Semantic Search for code - demo
+# Code search with Qdrant
 
-## Overview
+## Online version
 
-ToDo
+If you want to start with having a look at a running application, it is available here:
+**[https://code-search.qdrant.tech/](https://code-search.qdrant.tech/)**
 
-## How to run
+## Description
```

Comment:
I think you need a section on prerequisites. You've gained interest with the demo. This is the perfect place for them, something like:

```suggestion
## Prerequisites

To run this demo on your own system, install and/or set up the following components:

- `docker`
- `docker-compose`
- FastAPI
- ReactJS
- Rust parser project
- syn
- all-MiniLM-L6-v2
- microsoft/unixcoder-base
- A structured code base as a search database. For best results, you may need to refactor the code base

## Description
```

I've tried to organize all of the prerequisite packages into this intro section. @kacperlukawski , you'll need to add links... If you agree with my approach, once you've set up this list, you won't need other lines, such as "Please make sure to have Docker and Docker Composed installed your machine."

Reply:
@mjang Actually, having Docker, Docker Compose, Rust, and rust-analyzer is enough, as all the other components will be installed automatically when we compile/run the image. So we can make a list and remove that sentence, but it would only contain these four elements.

Reply:
@mjang `rustup` should be installed by the first line, so I would need to test it on a fresh environment. However, it's hard to predict all the system configurations, and I'm afraid we shouldn't cover the README into a detailed description for all the platforms. Can we leave it as it is?